src/capture/middlewares/header_middleware/packet_decoder.c

```diff
@@ -363,6 +363,10 @@ int extract_packets(const char *ltype, const struct pcap_pkthdr *header,
   cpac.length = header->len;
 
   os_strlcpy(cpac.ifname, interface, IF_NAMESIZE);
+
+  char id[MAX_RANDOM_UUID_LEN];
+  generate_radom_uuid(id);
+
   os_strlcpy(cpac.id, id, MAX_RANDOM_UUID_LEN);
```

Comment:
We can simplify to the following:

```suggestion
  generate_radom_uuid(cpac.id);
```